python/hsfs/feature_group.py

```diff
@@ -154,9 +160,26 @@ def insert(self, features, overwrite=False, storage=None, write_options={}):
             write_options,
         )
 
+        if self.statistics_config.enabled:
+            self._statistics_engine.compute_statistics(self, self.read(storage))
+
     def delete(self):
         self._feature_group_engine.delete(self)
 
+    def update(self):
+        self._feature_group_engine.update(self)
+        return self
```

Comment:
What is this method supposed to do?

Reply:
similarly to `.save()` and `.insert()` this is the method that triggers the update of metadata. It is only possible to update the description and statistics config of a feature group. The user can change these on the object and then call `.update()` to persist them.

Reply:
Ok